### PR TITLE
Added first skeleton README

### PR DESCRIPTION
**Pull request by ch-kr**
This PR adds the first draft of a somewhat descriptive README for this repo

### CODE REVIEW

**Review by ksamocha — APPROVED** [review]
Seems like a good high-level start to a README.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+* regional_missense_constraint/rmc/pipeline: Contains pipeline scripts that call utility functions to generate results.
+* regional_missense_constraint/rmc/pipeline/two_breaks: Folder that contains scripts to search for two simultaneous breaks; see `regional_missense_constraint/rmc/pipeline/two_breaks/README.md`.
+* regional_missense_constraint/rmc/utils: Contains utility functions that prepare input data or calculate results (RMC, missense badness, MPC).
+
+## Regional missense constraint
+Flowchart:
+![https://lucid.app/lucidchart/f359ebaa-5c40-4fa2-9641-01b7804751a2/edit?invitationId=inv_3ae908f2-7fba-43fa-b1a2-9369a1bd8a89#]
+
+Inputs:
+* Hail Table with all possible missense variants in the human exome (called VEP context)
```
Comment (lw453) [inline README.md:22]:
```suggestion
* Hail Table with all possible missense variants in the human exome (called the "VEP context" table)
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+* gnomAD v2.1.1 public sites Hail Table
+
+Output:
+* Hail Table annotated with RMC regions and filtered to curated transcripts; removed any transcripts following this [criteria](https://gnomad.broadinstitute.org/help/why-are-constraint-metrics-missing-for-this-gene-or-annotated-with-a-note)
+
+### Scripts:
```
Comment (lw453) [inline README.md:28]:
```suggestion
### Relevant scripts
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+* pipeline/regional_constraint.py: Pipeline code that prepares input Table, searches for first single/additional single breaks, and merges finalized results.
+* pipeline/two_breaks: See `regional_missense_constraint/rmc/pipeline/two_breaks/README.md` for explanation of scripts in this folder.
+* utils/generic.py: Utility functions that help prepare data for calculations.
+* utils/constraint.py: Utility functions that calculate RMC results.
```
Comment (lw453) [inline README.md:32]:
```suggestion 
| Path | Description | 
| :--- | :--- |  
| [`pipeline/regional_constraint.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/pipeline/regional_constraint.py) | Pipeline code that prepares input Table, searches for first single/additional single breaks, and merges finalized results. |
| [`pipeline/two_breaks`](https://github.com/broadinstitute/regional_missense_constraint/tree/main/rmc/pipeline/two_breaks) | Pipeline scripts to search for two simultaneous breaks; see [`README.md`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/pipeline/two_breaks/README.md). |  
| [`utils/generic.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/utils/generic.py) | Utility functions that help prepare data for calculations. | 
| [`utils/constraint.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/utils/constraint.py) | Utility functions that calculate RMC results. | 
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+
+## Missense badness
+### Scripts
+* pipeline/calculate_missense_badness.py: Pipeline code that prepares input Table and calculates missense badness score.
+* utils/missense_badness.py: Utility functions that prepare data for calculations and calculates missense badness.
```
Comment (lw453) [inline README.md:37]:
```suggestion
### Relevant scripts
| Path | Description | 
| :--- | :--- |  
| [`pipeline/calculate_missense_badness.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/pipeline/calculate_missense_badness.py) | Pipeline code that prepares input Table and calculates missense badness score. |
| [`utils/missense_badness.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/utils/missense_badness.py) | Utility functions that prepare data for calculations and calculate missense badness. | 
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,42 @@
...
+
+## MPC
+### Scripts
+* pipeline/calculate_mpc.py: Pipeline code that prepares input Table, calculates MPC, and annotates given input Table with MPC score.
+* utils/mpc.py: Utility functions that prepare data for calculations, creates MPC model regressions, and calculates MPC.
```
Comment (lw453) [inline README.md:42]:
```suggestion
### Relevant scripts
| Path | Description | 
| :--- | :--- |  
| [`pipeline/calculate_mpc.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/pipeline/calculate_mpc.py) | Pipeline code that prepares input Table, calculates MPC, and annotates given input Table with MPC score. |
| [`utils/mpc.py`](https://github.com/broadinstitute/regional_missense_constraint/blob/main/rmc/utils/mpc.py) | Utility functions that prepare data for calculations, create MPC model regressions, and calculate MPC. | 
```

**Pull request merged**